<launch>
	<node name="stage" pkg="stage_ros" type="stageros"
	      args="$(find astar)/world/playground.world" />
	<node name="planner_controller" pkg="astar" type="planner_node" /> 	
	<param name="goalX" value="4.5" />
	<param name="goalY" value="9.0" /> 
</launch>
